### DISCUSSION TITLE
Unexpected DIA-NN quit in 1.8.2

### DISCUSSION THREAD

**Discussion by sje-corbin** [vdemichev/DiaNN#805] [General]
Hi,

I have been having issues with the new version of DIANN (DIA-NN 1.8.2 beta 22) where the program quits during the search at the 'Quantification' step of searching a file just before the .quant file is written. While I have noticed that this often occurs repeatedly on specific files, sometimes restarting the search seems to overcome the issue, and the search will proceed with the file that encountered an error previously and complete without  the same issue occuring. Note, this is timsTOF diaPASEF data where I have this issue.

In cases where DIANN has repeated quit on the same file, I have since gone back and searched these on the previous version of DIANN and encountered no issues.

I am wondering if it is an issue with memory required, or if it has been documented previously, and if there is any advice regarding avoiding the error.

Cheers, Sam

**Comment by vdemichev** [comment]
Hi Sam,

All betas until 1.8.2 beta 27 contain serious bugs and can crash, we currently don't recommend using them.
The beta 27 does not have any known bugs so far, we are in the process of testing it in prepration for a release.

Best,
Vadim

**Comment by YuL-BI** [comment]
Hi Vadim,

Is there a command line for the retention time selection?  Thanks very much!

Best wishes,

Lu

**Reply by vdemichev** [reply]
Hi Lu,

What do you mean by RT selection?

Best,
Vadim

**Comment by YuL-BI** [comment]
Hi Vadim,

Well, I set runs too long so in the end of runs (~10min) there are quite a few very high peaks (1+ ions, contaminates) in the chromatographs. So I hope to exclude this section in the DIA-NN analysis.

Thank you very much,

Lu
________________________________

**Reply by vdemichev** [reply]
Hi Lu,

Excluding in the analysis itself is currently not possible (but if it were possible, would be very unlikely for it to noticeably affect the final results). What we did once in an experiment with poor hydrophobic fraction recovery in some runs was just exclude, when analysing the main DIA-NN report, all identifications with RT greater than a certain number. So this is what I would recommend, if you suspect they are not reliably quantified. 

Best,
Vadim

**Comment by YuL-BI** [comment]
Hi Vadim,

OK, I will try this.  Thank you very much.

Best wishes,

Lu
